pkg/unvanquished_src.dpkdir/ui/server_create.rml

```diff
@@ -60,7 +60,7 @@
 			</row>
 
 			<row id="maplist">
-				<dataselect source="mapList.default" fields="mapName" valuefield="mapLoadName" name="map" cvar="ui_dialogCvar1"/>
```

Comment:
isn't it a way to remember the selection?

Reply:
No, it doesn't do anything. Once this is merged, why don't you finish up #2883? 😛